go/storage/mkvs/interop/fixtures/consensus_mock.go

```diff
@@ -0,0 +1,52 @@
+package fixtures
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/oasisprotocol/oasis-core/go/common"
+	stakingInterop "github.com/oasisprotocol/oasis-core/go/consensus/tendermint/apps/staking/state/interop"
+	storage "github.com/oasisprotocol/oasis-core/go/storage/api"
+	"github.com/oasisprotocol/oasis-core/go/storage/mkvs"
+	db "github.com/oasisprotocol/oasis-core/go/storage/mkvs/db/api"
+	"github.com/oasisprotocol/oasis-core/go/storage/mkvs/node"
+)
+
+const consensusMockName = "consensus_mock"
+
+var consensusMockFixture = consensusMock{}
+
+type consensusMock struct {
+}
+
+func (c *consensusMock) Name() string {
+	return consensusMockName
+}
+
+func (c *consensusMock) Populate(ctx context.Context, ndb db.NodeDB) error {
+	var err error
+	testRoot := storage.Root{
+		Type:    storage.RootTypeState,
+		Version: 1,
+	}
+
+	mkvsTree := mkvs.New(nil, ndb, node.RootTypeState, mkvs.WithoutWriteLog())
+	if err = stakingInterop.InitializeTestStakingState(ctx, mkvsTree); err != nil {
+		return fmt.Errorf("consensus-mock: failed to initialize state: %w", err)
+	}
+	_, testRoot.Hash, err = mkvsTree.Commit(ctx, common.Namespace{}, 1)
+	if err != nil {
+		return fmt.Errorf("consensus-mock: failed to committ tree: %w", err)
+	}
+	if err = ndb.Finalize(ctx, []node.Root{testRoot}); err != nil {
+		return fmt.Errorf("consensus-mock: failed to finalize test root: %w", err)
+	}
+
+	fmt.Println("Consensus state root hash:", testRoot.Hash)
```

Comment:
This looks slightly hackish, maybe the `Populate` method should return the root and you can handle this in the caller for all fixtures?